hasher-matcher-actioner/webapp/src/Api.tsx

```diff
@@ -373,8 +373,9 @@ export async function deleteAction(name: string): Promise<{response: string}> {
 }
 
 // We need two different ActionRule types because the mackend model (must (not) have labels) is different
-// from the Frontend model (classification conditions)
-type APIActionRule = {
+// from the Frontend model (classification conditions).
+// This class should be kept in sync with python class ActionRule
+export type APIActionRule = {
```

Comment:
same feedback for this change as the one with `APIActionPerformer` 

Given `ActionRule` exists why are we using this one especially give what we return...